backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt

```diff
@@ -99,8 +104,12 @@ class CardMutationService {
         return CardActivationResultModel(ActivationState.success, encodedTotpSecret)
     }
 
-    private fun validateNewCard(card: CardGenerationModel): Boolean {
+    private fun validateCodeType(card: CardGenerationModel): Boolean {
         return (card.codeType == CodeType.STATIC && card.activationSecretBase64 == null) ||
             (card.codeType == CodeType.DYNAMIC && card.activationSecretBase64 != null)
     }
+
+    private fun validateCardHash(cardHash: ByteArray): Boolean {
+        return cardHash.size != 32
```

Comment:
We should probably add a comment here that we expect a SHA-256 hash, and thus the byte array should be of size 256/8=32.